metaci/cumulusci/views.py

```diff
@@ -123,10 +133,17 @@ def org_instance_delete(request, org_id, instance_id):
     instance.delete_org()
     return HttpResponseRedirect(instance.get_absolute_url())
 
-
-@staff_member_required
+@login_required
 def org_instance_detail(request, org_id, instance_id):
     instance = get_object_or_404(ScratchOrgInstance, org_id=org_id, id=instance_id)
+
+    # Verify access
+    try:
+        org = Org.objects.for_user(request.user).get(id=org_id)
```

Comment:
is there a reason we aren't just doing `request.user.has_perms('org_login', instance.org)` ?

Reply:
Because the user doesn't have org_login on the org itself.  They inherit that perm from the PlanRepository that's using the org.

Reply:
aha, I think that might come to be confusing later on (which is probably what @davisagli was getting at in his other comment). this is good for now but we probably want to move to permissions on org also, since PlanRepository and Org have a ...strange relationship, to say the least.